src/transformers/modeling_utils.py

```diff
@@ -2160,18 +2184,23 @@ def save_pretrained(
         # Checks if the model has been loaded in 8-bit
         if (
             getattr(self, "is_loaded_in_8bit", False)
-            and not getattr(self, "is_8bit_serializable", False)
-            and not _hf_peft_config_loaded
+            and getattr(self, "is_8bit_serializable", False)
+            and version.parse(importlib.metadata.version("bitsandbytes")) < version.parse("0.37.2")
         ):
```

Comment:
```suggestion
            and not _hf_peft_config_loaded
        ):
```
Why this has been removed?

Reply:
agree. perhaps lost it during another rebase